src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -267,3 +267,13 @@ def load_command_table(self, _):
             'aks_mesh_disable_ingress_gateway',
             supports_no_wait=True,
             confirmation=True)
+        g.custom_command(
+            'get-mesh-config',
+            'aks_mesh_get_mesh_config',
+            supports_no_wait=True
+        )
+        g.custom_command(
+            'migration-check',
+            'aks_mesh_migration_check',
+            supports_no_wait=True
+        )
```

Comment:
if you're curious about what this icon means, it's because it's generally good practice to end files with a newline. You can read more [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) if you're interested,but I think this summarizes it pretty well:
<img width="675" alt="image" src="https://github.com/sandesh714/azure-cli-extensions/assets/42152676/54605cde-3fc7-4300-b47e-17541d1d2d28">